### PR TITLE
Controller/d bvalidation

### PR DESCRIPTION
**Pull request by TheYisusGithub**

### CODE REVIEW

File: src/main/java/com/fundation/webservice/controller/Controller.java [hunk 1]
```diff
@@ -864,7 +876,7 @@ else if (configJson.getString("metadata").equals("xmp")) {
      * Endpoint for downloading zip file containing the products of a conversion process.
      *
      * @param fileName Name of the output zip file
```
Comment (alszla) [inline src/main/java/com/fundation/webservice/controller/Controller.java]: There's a missing point at  the end of the line.

File: src/main/java/com/fundation/webservice/common/Util.java [hunk 2]
```diff
@@ -68,23 +70,27 @@ private static void initUtil() {
...
-     * This method let me obtain the class Config for access to variables.
-     * @return Config.
+     * Method let me obtain the class Config for access to variables.
+     * @return Config the class, constains all variable required in DB connections.
      */
     public Config getConfig() {
         return config;
     }
-}
+}
```
Comment (alszla) [inline src/main/java/com/fundation/webservice/common/Util.java]: Please add an extra blank line.

File: src/main/java/com/fundation/webservice/database/DBConnection.java [hunk 3]
```diff
@@ -55,31 +57,24 @@ private static void initConnection() {
...
     /**
      * This method let me return the connections to db.
-     * @return conn.
+     * @return conn the variable tha contain the connection.
      */
     public Connection getConnection() {
         return conn;
     }
-}
+}
```
Comment (alszla) [inline src/main/java/com/fundation/webservice/database/DBConnection.java]: Add an extra blank line at the end.

Comment (TheYisusGithub) [reply src/main/java/com/fundation/webservice/database/DBConnection.java]: I fix that.

File: src/main/java/com/fundation/webservice/common/Util.java (hunk 2 above)
Comment (TheYisusGithub) [reply src/main/java/com/fundation/webservice/common/Util.java]: I fix that.

File: src/main/java/com/fundation/webservice/controller/Controller.java (hunk 1 above)
Comment (TheYisusGithub) [reply src/main/java/com/fundation/webservice/controller/Controller.java]: I fix that.

**Pull request merged**